Created orfFinder track

pwilx666 opened the pull request:
Created a new orfFinder track in jbrowseOrganismSpecificTracks, based on the orf tracks in jbrowse.dst files (which are currently commented out)

bgajria commented:
I think we need the "getJBrowseStyle" function in OrfTrackConfig.pm, as it is not doing anything additional.

pwilx666 commented:
I've removed the "getJBrowseStyle" function in OrfTrackConfig.pm and also fixed the local variables assigned to the addOrfs method in JbrowseOrgSpecificNaTracks.pm
